bhclub.html

```diff
@@ -44,6 +44,45 @@ <h1>BrickHack</h1>
         </section>
         <section id="about">
             <h1>About</h1>
+            <p>A hackathon is a “hacking marathon” in which teams or individuals have a specified time frame to create something from nothing. These events foster creative problem solving, teamwork, and innovative thinking within the hackathon community.</p>
+            <p>Our flagship event is BrickHack, a yearly 24-hour hackathon event. BrickHack attracts 600+ hackers from around the world each year. For more information about the next Brickhack event, check out <a href="./index.html" target="_blank">our event’s website</a>.</p>
+            <div class="card">
+                <button class="card-header">
+                    Do I need hackathon experience to join BrickHack?
+                    <i class="fa fa-plus" aria-hidden="true"></i>
+                </button>
+                <div class="card-body">
+                    <p>Nope! Having prior experience with hackathons is appreciated but not a requirement.</p>
+                </div>
+            </div>
+            <div class="card">
+                <button class="card-header">
+                    What will I be doing?
+                    <i class="fa fa-plus" aria-hidden="true"></i>
```

Comment:
It doesn't look like these are vertically centered :( 
I think it's noticeable enough to be worth adding `0.2rem` top margin or something; but this seems like a font issue that `display: inline-block` or `vertial-align: middle` or some other baseline adjustement could help fix

![image](https://user-images.githubusercontent.com/2782749/127256955-472a941f-bfda-40a0-9381-7186a1196013.png)